src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import path from 'ramda/src/path';
 import { Link } from 'react-router-dom';
-import { Grid, GridItemConstrainedMedium } from '#lib/styledGrid';
+import { GhostGrid, GridItemConstrainedMedium } from '#lib/styledGrid';
 import MetadataContainer from '../Metadata';
 import LinkedData from '../LinkedData';
+import text from '../Text';
+import Blocks from '../Blocks';
 import ATIAnalytics from '../ATIAnalytics';
 import cpsAssetPagePropTypes from '../../models/propTypes/cpsAssetPage';
 
+const componentsToRender = {
+  text,
+};
+
 const CpsAssetPageMain = ({ pageData }) => {
   const title = path(['promo', 'headlines', 'headline'], pageData);
   const summary = path(['promo', 'summary'], pageData);
   const metadata = path(['metadata'], pageData);
+  const blocks = path(['content', 'model', 'blocks'], pageData);
```

Comment:
Blocks component expects an array

```suggestion
  const blocks = path(['content', 'model', 'blocks'], pageData) || [];
```

Reply:
Done using pathOr :)